configs/peer/update_configs.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/sh
-# This script is intended for release updates,
-# when LTS and Stable branch configurations may change.
-# It downloads the previous configurations from raw.githubusercontent.com.
```

Comment:
this was wrong because docker image ID would  would be pointing to `hyperledger/iroha2:lts` but configs would be from previous lts. This is why when updating docker images we have to make sure that they are updated in this order exactly:
1. lts
2. stable
3. dev